BEP1.md

```diff
@@ -1,3 +1,11 @@
+<pre>
+  BEP: 1
+  Title: Purpose and Guidelines
+  Status: Living
+  Type: Process
+  Created: 2019-04-11
```

Comment:
Should we have version and modified time here?

Reply:
Currently, we do not have version or modified time for BEP yet.
Maybe it is not that necessary.